future_challenges_for_RSEng.tex

```diff
@@ -571,7 +575,7 @@ \section{What is in this future for RSEs?}
 %FG-NOTE: I think we can use this part to wrap things a bit up, and describe how this future then looks like.
 %GC-NOTE: Shall we add another story box here? Ideas what we write here?
 In the previous sections, we have tried to list some of the trends that we foresee.
-The material presented here is based on the outputs of a workshop, but extended with the authors' ideas.
```

Comment:
Not originating in this PR, but I guess we should bring the "extended with the authors' ideas" earlier, if not already there.